src/main/msp/msp_serial.c

```diff
@@ -132,11 +132,19 @@ static bool mspSerialProcessReceivedData(mspPort_t *mspPort, uint8_t c)
             }
             break;
 
-        case MSP_HEADER_START:  // Waiting for 'M' (MSPv1 / MSPv2_over_v1) or 'X' (MSPv2 native)
+        case MSP_HEADER_START:  // Waiting for 'M' (MSPv1 / MSPv2_over_v1) or 'X' (MSPv2 native) or STX (MSP CLI)
             mspPort->offset = 0;
             mspPort->checksum1 = 0;
             mspPort->checksum2 = 0;
             switch (c) {
+
```

Comment:
Is this really necessary? Why not &lt;enter-noninteractiove-cli&gt; MSP command? Or argument to existing CLI-enter

Reply:
Removed $ so there is no confusion that this is at all MSP.